mmedit/models/base_models/one_stage.py

```diff
@@ -144,7 +150,8 @@ def forward(self, inputs, data_samples, mode='tensor'):
         else:
             raise ValueError('Invalid forward mode.')
 
-    def train_step(self, data: List[dict], optim_wrapper):
+    def train_step(self, data: List[dict],
+                   optim_wrapper: Dict[str, torch.optim.Optimizer]) -> dict:
```

Comment:
`optim_wrapper` seems to be an`OptimWrapperDict`. I think you can add `from mmengine.optim import OptimWrapperDict` at the start of this file and then use `OptimWrapperDict` as typing hint.